lib/games/addDeveloperProduct.js

```diff
@@ -38,8 +39,8 @@ const nextFunction = (jar, token, universeId, name, priceInRobux, description) =
         return json
       }
       throw new Error(json)
```

Comment:
Sorry just been reading through and spotted this
- Would rather avoid throwing and immediately catching errors 
- This will throw a JSON parsing error as an API error, which it isn't necessarily
Could you please refactor to separate the JSON parsing error (& associated try-catch) and the API error handling?